src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,155 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static class Node<T> {
+        T element;
+        Node<T> next;
+        Node<T> prev;
+
+        public Node(T element, Node<T> next, Node<T> prev) {
+            this.element = element;
+            this.next = next;
+            this.prev = prev;
+        }
+    }
```

Comment:
move this class to the very end of your code

Reply:
done